test/e2e_test.go

```diff
@@ -273,6 +273,14 @@ var _ = Describe("e2e", func() {
 			printGrpcError(err)
 			Expect(err).NotTo(HaveOccurred(), "commit chaincode")
 
+			result, err := chaincode.QueryCommitted(specCtx, peerConnections[0].connection, peerConnections[0].id, channelName)
```

Comment:
The commit above should have caused the chaincode to be committed on all network peers where it was installed. We could confirm this by querying installed chaincode on both org peers.

**However**, we only know for sure that the commit has completed on the (Gateway) peer where we invoked the commit since that is where we observe the successfully committed transaction in the ledger. The other peers may not yet have processed the transaction. So I think this approach of querying just the peer used for the commit is reasonable.

One thing I would change though... using `peerConnections[0]` happens to be the same peer that was used for the commit but that could change if somebody changed the code above later. I think it would be better to be more explicit about the peer being used with `n_conn1` as the connection and `org1MSP` as the identity.

Reply:
Thanks for the pointer, @bestbeforetoday. Changing it now.